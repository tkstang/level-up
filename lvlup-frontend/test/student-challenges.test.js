import StudentChallenges from '../src/components/Student/challenges/challenges-main';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

test('StudentChallenges should render', () => {
  const challenge = shallow(
    <StudentChallenges />,
  );
  expect(toJson(challenge)).toMatchSnapshot();
});
