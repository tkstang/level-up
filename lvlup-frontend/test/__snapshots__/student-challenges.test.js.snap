exports[`test StudentChallenges should render 1`] = `
<lifecycle(StudentChallengesTable)
  campusChallenges={[Function]}
  resetChallenge={[Function]}
  selectChallenge={[Function]}
  sortChallengeCategory={[Function]}
  sortChallengeCategoryReverse={[Function]}
  sortChallengeName={[Function]}
  sortChallengeNameReverse={[Function]}
  sortChallengePoints={[Function]}
  sortChallengePointsReverse={[Function]}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
  storeSubscription={
    Subscription {
      "listeners": Object {
        "notify": [Function],
      },
      "onStateChange": [Function],
      "parentSub": undefined,
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "unsubscribe": null,
    }
  } />
`;
